tests/tests_integration/tests_diff_version/tests_common/test_commands.py

```diff
@@ -0,0 +1,29 @@
+from th2_data_services import Data
+
+def test_break_iteration_get_events(demo_events_from_data_source: Data):
```

Comment:
1. I think that we have to check it for both
 - http DS 
 - grpc DS
 so `demo_events_from_data_source` should be a fixture that returns Data generated by both of them.